stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/DeletedMessageListItemPredicate.kt

```diff
@@ -29,7 +30,9 @@ public sealed class DeletedMessageListItemPredicate : MessageListView.MessageLis
      */
     public object VisibleToAuthorOnly : DeletedMessageListItemPredicate() {
         override fun predicate(item: MessageListItem): Boolean {
-            return true
+            return ChatClient.instance().getCurrentUser()?.let { user ->
```

Comment:
We could do a small optimization and firstly check if `item is MessageListItem.MessageItem` and only then get the current user and compare it with `item.message.user.id`

Reply:
This is already happening, the language does it for us. If the first part of a boolean expression fails, it stops checking th rest, if it is an and (&&). So if the `item is MessageListItem.MessageItem` fails, the second part won't be checked.